### ISSUE TITLE
Unable to use Pojomatic with Java 9

### ISSUE THREAD

**Issue by StephaneArguin** [irobertson/pojomatic#10]
ASM 5.x doesn't support Java 9 (cause IllegalArgumentException). Need to use ASM 6.0 : 

```
<dependency>
  <groupId>org.ow2.asm</groupId>
  <artifactId>asm</artifactId>
  <version>6.0</version>
</dependency>
```

**Comment by karteeksaragad** [comment]
I don't think we can use ASM 6.0 with this pojomatic 2.0.1.  
**Reason**:  pojomatic-2.0.1 is tightly dependent on org.kohsuke.asm5 (compile dependency). pojomatic/pojomatic/src/main/java/org/pojomatic/internal/LocalVariable.java and few other classes are using the below imports... 
import org.kohsuke.asm5.Label;
import org.kohsuke.asm5.MethodVisitor;
import org.kohsuke.asm5.Opcodes;
import org.kohsuke.asm5.Type;

We have tried excluding "org.kohsuke.asm5" dependency from "org.pojomatic:2.0.1" and added the below dependency... it didn't work as it is a compilation dependency.

<dependency>
  <groupId>org.ow2.asm</groupId>
  <artifactId>asm</artifactId>
  <version>6.0</version>
</dependency>

We are waiting for the pojomatic guys to release their new java9 compatible plugin soon.

Thanks,
Karteek.

**Comment by irobertson** [comment]
Fixed in release 2.1.0
